### DISCUSSION TITLE
Modification/Extension/Clarification on the Author part of Packages.xml

### DISCUSSION THREAD

**Discussion by NikRimington** [umbraco/Umbraco.Packages#71] [General] [answered]
Based on a discussions with Matt Brailsford, it would be good if the Author part of Packages.xml could support multiple authors if it doesn't already.

If it does, we need to document how it works.

**Comment by KevinJump** [comment]
Based on the Umbraco Source, there can only be one Author in the XML .
(see https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Core/Models/Packaging/PackageDefinition.cs)

You can have multiple Contributors - but there is only ever one 'author'

**Comment by jmayntzhusen** [comment]
So to support this we'd have to change it in the CMS, it would of course not be backwards compatible, and we'd probably need to change a few things on Our and in UmbPack as well.

Guess the big question here is why is this necessary? Could be a candidate for change from the next major version for example, but does it make sense?

I will transfer this to a discussion and open it up for people to comment on.

**Comment by ronaldbarendse** [comment, marked as answer]
Since we're using NuGet packages for v9 and up, you can specify multiple authors (using a comma or semi-colon separated list, depending on whether it's specified as MSBuild or NuSpec property), see: https://learn.microsoft.com/en-us/nuget/create-packages/package-authoring-best-practices#package-metadata.
